platform/mellanox/issu-version.mk

```diff
@@ -5,6 +5,4 @@ $(ISSU_VERSION_FILE)_SRC_PATH = $(PLATFORM_PATH)/issu-version
 $(ISSU_VERSION_FILE)_DEPENDS += $(APPLIBS)
 SONIC_MAKE_FILES += $(ISSU_VERSION_FILE)
 
-MLNX_FILES += $(ISSU_VERSION_FILE)
-
```

Comment:
Removes from MLNX_FILES which is a target executed in the host build environment (bullseye). We set this as a dependency of syncd in order to have it build on buster.

Reply:
This is added for ssd fw update tool. I am not sure the effects of removing it here.

Reply:
MLNX_FILES does not have anything to do with syncd, it is used to put files onto the host. Because I cannot build issu-verison in bullseye I cannot use MLNX_FILES as the target for it, I need to use something tied to a docker I know the SDK will be available in (syncd). That is why it is removed from MLNX_FILES. However it is installed just the same on the host so anything with a dependency on it will not have a problem.